src/Stratis.Bitcoin.Features.Consensus.Tests/TimeSyncBehaviorAdjustedTimeTest.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Net;
+using Stratis.Bitcoin.Base;
+using Stratis.Bitcoin.Utilities;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests
+{
+    /// <summary>
+    /// Tests of <see cref="TimeSyncBehavior"/> and <see cref="TimeSyncBehaviorState"/> classes.
+    /// </summary>
+    public partial class TimeSyncBehaviorAdjustedTimeTest
+    {
+        /// <summary>
+        /// Time data is added this state. 
+        /// </summary>
+        private TimeSyncBehaviorState timesyncBehaviourState;
+
+        /// <summary>
+        /// This provides the adjusted time ready for assertion. 
+        /// </summary>
+        private IDateTimeProvider dateTimeProvider;
+        
+        /// <summary>
+        /// This is the result of the operation under test. 
+        /// </summary>
+        private TimeSpan offset;
+
+        /// <summary>
+        /// Checks that <see cref="TimeSyncBehaviorState.AddTimeData(IPAddress, TimeSpan, bool)"/>
+        /// uses outbound samples as a priority over inbound as they are less likely to be malicious.
+        /// This means that even when there are many inbound connections, only the first ones are considered until there are enough outbound nodes. 
+        /// Eg only 9 inbound per 1 outbound.
```

Comment:
use hmldoc-see to reference the constant here

Reply:
yes agree, I'll do that